Project.toml

```diff
@@ -8,12 +8,14 @@ Catalyst = "479239e8-5488-4da2-87a7-35f2df7eef83"
 HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
+SymbolicUtils = "d1185830-fcd6-423d-90d6-eec64667417b"
 
 [compat]
 Catalyst = "13"
 HTTP = "1.9.6"
 JSON = "0.21.4"
 JSON3 = "1.13.1"
+SymbolicUtils = "3.5"
```

Comment:
This package already realies on this indirectly.
And pretty heavily so, as i understand the metadata system.